packages/cli/commands/project/dev.js

```diff
@@ -253,9 +253,9 @@ exports.handler = async options => {
     );
   }
 
-  if (result && result.error) {
+  if (!initialUploadResult || initialUploadResult.uploadError) {
```

Comment:
If `initialUploadResult` is undefined, do we need to account for that in this condition? Will this error out if `initialUploadResult.uploadError` is undefined?

Reply:
Good call, I added some edge case handling in [5e8e5be](https://github.com/HubSpot/hubspot-cli/pull/877/commits/5e8e5be8f83f0dcab2cd305559c756fd4c367540)